GetandTake/Core/Filters/LogActionFilterAttribute.cs

```diff
@@ -0,0 +1,37 @@
+﻿using GetandTake.Configuration.Settings;
+using Microsoft.AspNetCore.Mvc.Filters;
+
+namespace GetandTake.Core.Filters;
+
+public class LogActionFilterAttribute : ResultFilterAttribute, IAsyncPageFilter
+{
+    private readonly ILogger<LogActionFilterAttribute> _logger;
+    private readonly AppSettings _appSettings;
+
+    public LogActionFilterAttribute(
+        ILogger<LogActionFilterAttribute> logger,
+        AppSettings appSettings)
+    {
+        _logger = logger;
+        _appSettings = appSettings;
+    }
+
+    public async Task OnPageHandlerExecutionAsync(
+        PageHandlerExecutingContext context,
+        PageHandlerExecutionDelegate next)
+    {
+        var page = context.RouteData.Values["page"];
+
+        if (_appSettings.LogFilter.IsLogFilterActive == true)
```

Comment:
You can just simplify it:
```suggestion
        if (_appSettings.LogFilter.IsLogFilterActive)
```